adafruit_ble/__init__.py

```diff
@@ -167,7 +213,14 @@ def stop_scan(self):
         self._adapter.stop_scan()
 
     def connect(self, advertisement, *, timeout=4):
-        """Initiates a `BLEConnection` to the peer that advertised the given advertisement."""
+        """
+        Initiates a `BLEConnection` to the peer that advertised the given advertisement.
+
+        :param advertisement Advertisement: An `Advertisement` or a subclass of `Advertisement`
+        :param timeout float: how long to wait for a connection
+        :returns the connection to the peer
+        :rtype BLEConnection
```

Comment:
This work? I thought it should ble `:returns:`.

Reply:
You're right. I find Sphinx to be very non-obvious.
https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/develop/libraries/Bluefruit52Lib/examples/Peripheral/cplay_ble/services.txt

Also the `rtype` must be specified separately from `return` or `returns`, unlike `param`.

Reply:
Also a `:return:` for a constructor is kinda obvious. I removed that one.